### PR TITLE
Discount-pipe_CU-md8j9h

### PR DESCRIPTION
**Pull request by a-wilkes**

### CODE REVIEW

**Comment by a-wilkes** [comment]
Task linked: [CU-md8j9h Discount pipe](https://app.clickup.com/t/md8j9h)

**Comment by lpboucher** [comment]
Task linked: [CU-md8j9h Discount pipe](https://app.clickup.com/t/md8j9h)

File: src/app/shared/pipes/volume-discount.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Pipe, PipeTransform } from "@angular/core";
+
+import { PricingTierService } from "@core/pricing/pricing-tier.service";
+
+@Pipe({
+    name: "volumeDiscount"
+})
+export class VolumeDiscountPipe implements PipeTransform {
+    private static ACTIVE_DISCOUNT_REDUCTION: number = 0.55;
+    private static INACTIVE_DISCOUNT_REDUCTION: number = 0.7;
```
Comment (lpboucher) [inline src/app/shared/pipes/volume-discount.pipe.ts]: prefer constant variables outside class

Comment (lpboucher) [reply src/app/shared/pipes/volume-discount.pipe.ts]: done

**Review by lpboucher — APPROVED** [review]

**Pull request merged**